lib/ModuleNotFoundError.js

```diff
@@ -36,7 +36,6 @@ const previouslyPolyfilledBuiltinModules = {
 	tty: "tty-browserify",
 	url: "url",
 	util: "util",
-	vm: "vm-browserify",
```

Comment:
Please keep this line. It’s a hint to tell the users how to polyfill node builtin modules.